docs/xxprintflagsfinal.md

```diff
@@ -0,0 +1,40 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]PrintFlagsFinal
+
+When enabled, this Hotspot option outputs the values of an essential subset of configuration parameters.
```

Comment:
"essential" doesn't carry the right meaning to me....I would expect a much larger list of options to be included in an "essential" list, but I know what you mean. I can't come up with a better word or phrase at the moment :( .

I'm also not sure we should call it a "Hotspot" option here, unless that's consistent with any other option we implement for Hotspot compatibility....Maybe instead the sentence could end with "in a format compatible with that produced by Hotspot" ?

Reply:
..."options we have been told or found to be expected by various software projects and packages?"

sounds awkward, but maybe you can find a cleaner way to say that?